tests/system/Autoloader/FileLocatorTest.php

```diff
@@ -119,4 +120,100 @@ public function testLocateFileReturnsEmptyWithBadNamespace()
 	}
 
 	//--------------------------------------------------------------------
+        
+        public function testSearchSimple()
+	{
+                $expected = rtrim(APPPATH,'/').'/Views/form.php';
```

Comment:
I was working on my windows machine, so without that test would fail under my setup.